modules/library/src/main/scala/zio/elasticsearch/request/package.scala

```diff
@@ -20,26 +20,85 @@ import zio.elasticsearch.query.sort.Sort
 
 package object request {
   private[elasticsearch] trait HasRefresh[R <: HasRefresh[R]] {
+
+    /**
+     * Configures whether or not to refresh the index after the operation in the [[ElasticRequest]].
+     *
+     * @param value
+     *   `true` if the index should be refreshed after the operation, `false` otherwise
+     * @return
+     *   returns a new instance of the [[ElasticRequest]] with the `refresh` value set.
+     */
     def refresh(value: Boolean): R
 
+    /**
+     * Sets `refresh` parameter to `false` in the [[ElasticRequest]].
+     *
+     * @return
+     *   returns a new instance of the [[ElasticRequest]] with the `refresh` value set to `false`.
+     */
     final def refreshFalse: R = refresh(false)
 
+    /**
+     * Sets `refresh` parameter to `true` in the [[ElasticRequest]].
+     *
+     * @return
+     *   returns a new instance of the [[ElasticRequest]] with the `refresh` value set to `true`.
+     */
     final def refreshTrue: R = refresh(true)
   }
 
   private[elasticsearch] trait HasRouting[R <: HasRouting[R]] {
+
+    /**
+     * Specifies a `routing` value to be used for this [[ElasticRequest]].
+     *
+     * @param value
+     *   the [[Routing]] value to set for the [[ElasticRequest]]
+     * @return
+     *   returns a new instance of the [[ElasticRequest]] with the `routing` value set.
+     */
     def routing(value: Routing): R
   }
 
   private[elasticsearch] trait HasFrom[R <: HasFrom[R]] {
+
+    /**
+     * Sets the starting offset from where the [[zio.elasticsearch.ElasticRequest.SearchRequest]] or the
+     * [[zio.elasticsearch.ElasticRequest.SearchAndAggregateRequest]] should return results.
+     *
+     * @param value
+     *   the `Int` value to set the `from` parameter in the [[ElasticRequest]]
+     * @return
+     *   returns a new instance of the [[ElasticRequest]] type with the `from` parameter set.
```

Comment:
Maybe to use `an instance of the [[ElasticRequest]] enriched with the `from` parameter.